docs/user-guide/src/bmo/live-iso.md

```diff
@@ -0,0 +1,32 @@
+# Live-ISO
+
+The live-iso API in Metal3 allows to boot BaremetalHost with live-iso image instead of using IPA deploy ramdisk.
+
+Why we need it?
+
+In some circumstances, i.e to reduce boot time for ephemeral workloads, it may be possible to boot an iso
+and not deploy any image to disk (saving the time to write the image and reboot). Since this alternative
+is distributed as a live-iso, leveraging the existing toolchains like [fedora-coreos installer](https://docs.fedoraproject.org/en-US/fedora-coreos/bare-metal/) might be desirable.
+
+How to use it?
+
+Here is an example with a BaremetalHost CRD, where iso referenced by the url and `live-iso` set in DiskFormat will be
+live-booted without deploying an image to disk. Checksum options are not required in this case, and will be ignored if specified:
+
```

Comment:
An important bit is missing: the live ISO mode is only supported for virtual media drivers.

Reply:
Good point, added now.